### ISSUE TITLE
_params == null

### ISSUE THREAD

**Issue by hadi-14** [vishnuagbly/google_sign_in_all_platforms#10]
![image](https://github.com/user-attachments/assets/053d9026-4f2f-43bd-8ad2-cad574d55ea9)

Hey Dev, I can log in the first time and if I log out again, I get this error until I close the app from the background and log in again. I am initializing on the login page.

**Comment by vishnuagbly** [comment]
Are you seeing this with example app as well?

**Comment by devrajraut** [comment]
I'm facing the same Issue no matter what I do. Maybe I'm implementing the function wrong. Could you please give a short code of the signInWithGoogle() async {} function.

Error during Google sign-in: 'package:google_sign_in_all_platforms_interface/google_sign_in_all_platforms_interface.dart': Failed assertion: line 47 pos 12: '_params == null': __params is already not null

Kindly Help. Thank you.

**Comment by vishnuagbly** [comment]
Got it, can you remind me to look into it, after 3-4 hours?

**Comment by vishnuagbly** [comment]
Wait better recommend after 6 hours

**Comment by devrajraut** [comment]
Hi. Reminder to solve my query and Help me Pls. Thanks.

**Comment by vishnuagbly** [comment]
Did you try running the example code?

**Comment by devrajraut** [comment]
Yes. Like I said. I am able to Login and Logout. But the Issue is when I Logout and try to Login. The Error comes up as the GoogleSignInParams is called again.

**Comment by devrajraut** [comment]
Any luck with this Yet? Awaiting your Response. Thanks so much for this wonderful Package. I just hope it works for me.

**Comment by vishnuagbly** [comment]
Ohh Okk... Actually have been a little busy, if you already have the setup
handy, can you try to debug and find the issue?

**Comment by devrajraut** [comment]
Writing to you was my last resort. I have been breaking my head over this for the past 2 days. Please when u get the time, resolve this. This issue is open since Aug 2024. Im assuming more people have had this issue. If anyone else knows anything, please respond. Thanks..

**Comment by vishnuagbly** [comment]
I see.. let me look into it then. This always happens with example app?
When we login logout then re-login?

**Comment by vishnuagbly** [comment]
Hi @devrajraut @hadi-14 I cannot seem to reproduce the issue, can you provide the platform you are working on, and if possible a Minimum Reproducible Code Example.

**Comment by vishnuagbly** [comment]
I see..., Once, I have verified, I will either add a "Note" for this in the README or if fixable add a fix.
Thank You @kezza2k7

**Comment by vishnuagbly** [comment]
For now I am closing this, if anyone faces this issue, even after following @kezza2k7 mentioned fix. We can reopen this.
